src/components/claim-paged.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+import { Button } from "./button";
+
+import { Links } from "../constants/links";
+
+export function ClaimPaged() {
+  return (
+    <section className="md:py-20 py-8 bg-shutter-new">
+      <div className="mx-auto md:container px-8">
+        <div className="">
+          <h1 className="text-4xl md:text-6xl leading-tight text-white">
+            Introducing the Shutter Network Genesis Allocations
```

Comment:
Everywhere shutter network was changed to just Shutter. Is it intended to leave Shutter network here?